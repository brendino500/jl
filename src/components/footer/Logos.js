import { makeStyles } from '@material-ui/core/styles'

import xbox from '../../assets/xbox-logo.png'
import ps4 from '../../assets/ps4-logo.png'
import steam from '../../assets/steam-logo.png'
import pcsoftware from '../../assets/pc-software-logo.png'
import rating from '../../assets/age-rating-logo.png'
import selogo from '../../assets/se-logo-black.png'

const useStyles = makeStyles((theme) => ({
  logoContainer: {
    display: 'flex',
    flexDirection: 'column',
  },
  logoGrid: {
    marginTop: '3%',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
    [theme.breakpoints.between('xs', 'sm')]: {
      flexWrap: 'wrap',
    },
  },
  logos: {
    height: '3em',
    marginLeft: '2em',
  },
  seLogo: {
    height: '2em',
    width: 'auto',
    marginTop: '2em',
    marginBottom: '1em',
    [theme.breakpoints.between('xs', 'sm')]: {
      height: 20,
    },
  },
  seLogoContainer: {
    display: 'flex',
    justifyContent: 'center',
  },
}))

export default function Logos() {
  const classes = useStyles()

  const logos = [
    { image: rating, name: 'rating-logo' },
    { image: xbox, name: 'xbox-logo' },
    { image: ps4, name: 'ps4-logo' },
    { image: steam, name: 'steam-logo' },
    { image: pcsoftware, name: 'pcsoftware-logo' },
  ]
  return (
    <div className={classes.logoContainer}>
      <div className={classes.logoGrid}>
        {logos.map((logo, index) => (
          <img
            key={index}
            src={logo.image}
            alt="logo"
            className={`${logo.name}`}
          />
        ))}
      </div>
      <div className={classes.seLogoContainer}>
        <img src={selogo} alt="se logo" className={classes.seLogo} />
      </div>
    </div>
  )
}
