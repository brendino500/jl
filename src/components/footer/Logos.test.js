import renderer from 'react-test-renderer'
import Logos from './Logos'

it('Logos matches snapshot', () => {
  const tree = renderer.create(<Logos />).toJSON()
  expect(tree).toMatchSnapshot()
})
