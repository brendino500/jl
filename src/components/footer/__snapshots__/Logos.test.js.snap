// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logos matches snapshot 1`] = `
<div
  className="makeStyles-logoContainer-1"
>
  <div
    className="makeStyles-logoGrid-2"
  >
    <img
      alt="logo"
      className="rating-logo"
      src="age-rating-logo.png"
    />
    <img
      alt="logo"
      className="xbox-logo"
      src="xbox-logo.png"
    />
    <img
      alt="logo"
      className="ps4-logo"
      src="ps4-logo.png"
    />
    <img
      alt="logo"
      className="steam-logo"
      src="steam-logo.png"
    />
    <img
      alt="logo"
      className="pcsoftware-logo"
      src="pc-software-logo.png"
    />
  </div>
  <div
    className="makeStyles-seLogoContainer-5"
  >
    <img
      alt="se logo"
      className="makeStyles-seLogo-4"
      src="se-logo-black.png"
    />
  </div>
</div>
`;
